<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>835</width>
    <height>622</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QComboBox" name="comboBox"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QLabel" name="DisplayLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>1</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
          <property name="cursor">
           <cursorShape>CrossCursor</cursorShape>
          </property>
          <property name="mouseTracking">
           <bool>false</bool>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">border-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QPushButton" name="EnumButton">
            <property name="text">
             <string>查找设备</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="OpenButton">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>打开设备</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="CloseButton">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>关闭设备</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QRadioButton" name="ContinuesRadioButton">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="cursor">
               <cursorShape>ArrowCursor</cursorShape>
              </property>
              <property name="text">
               <string>连续模式</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="TriggerModeRadio">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>触发模式</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QPushButton" name="StartGrabbingButton">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>开始采集</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="StopGrabbingButton">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>停止采集</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QCheckBox" name="SoftwareTriggerCheck">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>软触发</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="SoftwareOnceButton">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>软触发一次</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QPushButton" name="SaveBmpButton">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>保存BMP</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="SaveTiffButton">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>保存tiff</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QPushButton" name="SaveJpgButton">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>保存JPG</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="SavePngButton">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>保存png</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="label">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>曝光</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="ExposureLineEdit">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>增益</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="GainLineEdit">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>帧率</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="FrameRateLineEdit">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <widget class="QPushButton" name="GetParamButton">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>参数获取</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="SetParamButton">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>设置参数</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>835</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
